### PR TITLE
UI Updates to Results Lookup 

### PR DESCRIPTION
**Pull request by akshaaatt**
- UI Updates to ArtistBio, ReleaseList, ReleaseInfo, Links and UserData.
- Added Lottie Animation View and a few assets.
- Added attributions.

### CODE REVIEW

File: app/src/main/res/layout/fragment_bio.xml [hunk 1]
```diff
@@ -1,30 +1,110 @@
...
+        <com.airbnb.lottie.LottieAnimationView
+            android:id="@+id/animationView"
+            android:layout_above="@+id/cl"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            app:lottie_autoPlay="true"
+            app:lottie_loop="true"
+            app:lottie_rawRes="@raw/group"/>
+
+        <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (amCap1712) [inline app/src/main/res/layout/fragment_bio.xml:26]: A `ConstraintLayout` layout inside a `RelativeLayout` seems weird. Let's try to flatten the hirerachy by moving the `LottieAnimationView` inside ConstraintLayout and removing the RelativeLayout.

File: app/src/main/res/layout/card_release_info.xml [hunk 2]
```diff
@@ -1,77 +1,82 @@
 <?xml version="1.0" encoding="utf-8"?>
-
-<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (amCap1712) [inline app/src/main/res/layout/card_release_info.xml:2]: Let's use a ConstraintLayout :). Also, can we get rid of the LinearLayout inside it?

**Review by amCap1712 — CHANGES_REQUESTED** [review]
Looks good, some more changes and we'll be ready to merge.

File: app/src/main/res/layout/fragment_bio.xml (hunk 1 above)
Comment (akshaaatt) [reply app/src/main/res/layout/fragment_bio.xml:26]: Haha! I guess my habit of using RelativeLayout needs to be flushed XD
Sure I'll fix this soon.

Comment (akshaaatt) [reply app/src/main/res/layout/fragment_bio.xml:29]: Done

File: app/src/main/res/layout/card_release_info.xml (hunk 2 above)
Comment (akshaaatt) [reply app/src/main/res/layout/card_release_info.xml]: Done boss

**Comment by akshaaatt** [comment]
All done! :)

File: app/src/main/res/layout/fragment_bio.xml (hunk 1 above)
Comment (amCap1712) [reply app/src/main/res/layout/fragment_bio.xml:29]: Why do we need another ConstraintLayout here? Is it possible to just keep the outermost ConstrainLayout and remove this one?

Comment (akshaaatt) [reply app/src/main/res/layout/fragment_bio.xml:29]: I had grouped all the artist information that we get from our API and highlighted them uniquely to separate it from the wiki stuff.

Comment (akshaaatt) [reply app/src/main/res/layout/fragment_bio.xml:29]: This second ConstraintLayout is like a container to fit all the artist information from our API in one place.

Comment (akshaaatt) [reply app/src/main/res/layout/fragment_bio.xml]: Removed the extra ConstraintLayout

**Review by amCap1712 — APPROVED** [review]
Changes look good! Thanks

**Pull request merged**